docs/src/data/power.js

```diff
@@ -96,4 +96,10 @@ export const displayPower = {
     active: 10000, // Estimated power draw when about half the pixels are on
     sleep: 7, // Deep sleep power draw (display off)
   },
+  // Based on the nice!view using Sharp's LS011B7DH01
+  NICEVIEW: {
+    activePercent: 0.01, // Estimated two refreshes per second taking five milliseconds each
```

Comment:
Why 2 refresh/s?

Reply:
To be honest, there's not an exact reason this was chosen. The ePaper estimate only had one change per minute taking three seconds, which maybe was a bit low for normal keyboard use. The view can refresh very quickly (5ms), so the active current is only that high for that short period. Because of this, if I had only done one change per minute, the percentage would've been really low. I think an active percent of 1% is applicable for the view and maybe even overestimates how much power it uses.